recipe/meta.yaml

```diff
@@ -28,21 +28,29 @@ requirements:
     - pip
     - hatchling >=1.12.2
     - hatch-vcs >=0.2.0
-    - importlib_resources  # [py<39]
   run:
     - python
     - conda >=23.9.0
     - pip >=23.0.1
     - grayskull
-    - importlib_resources  # [py<39]
     - packaging
+    - unearth
+    - pydantic
+    - python-build
+    - python-installer
+    - requests
+    - ruamel.yaml
+    - typer
+    - platformdirs
+    - conda-index
+    - conda-libmamba-solver
```

Comment:
I'm not sure if conda-libmamba-solver is required here as an explicit run dependency, since it is implicitly included with conda.